pkg/repositories/gormimpl/task_repo.go

```diff
@@ -53,6 +50,9 @@ func (r *TaskRepo) Get(ctx context.Context, input interfaces.Identifier) (models
 			Version: input.Version,
 		})
 	}
+	if tx.Error != nil {
+		return models.Task{}, r.errorTransformer.ToFlyteAdminError(tx.Error)
```

Comment:
Would be good to cover these error paths if its not too much effort through unit tests

Reply:
I'm working on it